backend/tests/prisonerPersonal.test.ts

```diff
@@ -2289,5 +2299,28 @@ describe('prisoner personal', () => {
         expect.objectContaining({ displayNeurodiversity: false })
       )
     })
+
+    it('should return undefined if prisoner not in users caseload', async () => {
+      const neurodivergence = {
+        prn: 'ABC123',
+        establishmentId: 'MDI',
+        establishmentName: 'Moorland (HMP & YOI)',
+        neurodivergenceSelfDeclared: NeurodivergenceSelfDeclared.Autism,
+        selfDeclaredDate: '10 February 2022',
+        neurodivergenceSupport: [NeurodivergenceSupport.Reading, NeurodivergenceSupport.AuditorySupport],
+        supportDate: '14 February 2022',
+      }
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      esweService.getNeurodivergence = jest.fn()
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      curiousApi.getLearnerNeurodivergence.mockResolvedValue(neurodivergence)
+      await controller(req, res)
+      expect(res.render).toHaveBeenCalledWith(
+        'prisonerProfile/prisonerPersonal/prisonerPersonal.njk',
+        expect.objectContaining({ neurodivergence: undefined })
```

Comment:
Just to confirm - is `undefined` ok here?

Reply:
Yes - when prisoner is not in accelerated prison

Reply:
I know, but in https://github.com/ministryofjustice/digital-prison-services/blob/ESWE-287_feature_flag_neurodivergence_indicator_repair/views/prisonerProfile/prisonerPersonal/partials/neurodiversities.njk#L3 it references a property in that value: `{% set neurodiversityInfo = neurodivergence.content %}` - does that not cause a rendering error?